bin/orchestrator-entrypoint.sh

```diff
@@ -12,128 +12,157 @@ ENVIRONMENT="DJANGO_SETTINGS_MODULE=settings.dev"
 
```

Comment:
Should an entry about this PR fix been added to CNR? (although not sure if this repository use CNR file to have inter-releases change log...)